can/util.py

```diff
@@ -211,7 +211,16 @@ def _create_bus_config(config: Dict[str, Any]) -> typechecking.BusConfig:
         raise CanInterfaceNotImplementedError(
             f'Unknown interface type "{config["interface"]}"'
         )
-
+    if "port" in config:
+        if isinstance(config["port"], str):
+            if config["port"].isnumeric():
+                config["port"] = int(config["port"])
+            else:
+                raise ValueError("Port config must be a number!")
+        elif not isinstance(config["port"], int):
+            raise TypeError("Port config must be string or integer!")
+    if not 0 < config["port"] < 65535:
+        raise ValueError("Port config must be inside 0-65535 range!")
```

Comment:
This needs to be inside the `if "port" in config:` block

Reply:
oh, thank you, I'm an idiot😅